libraries/HID/src/HID.cpp

```diff
@@ -133,25 +133,48 @@ bool HID_::setup(USBSetup& setup)
 		}
 		if (request == HID_SET_REPORT)
```

Comment:
Is keyboard LEDs the only time this kind of request is being actively used? I'd prefer to add some more generic API as in PluggableUSB framework